source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/http/HttpRequestResult.java

```diff
@@ -26,15 +32,16 @@ public RESULT getResult()
 
     public Exception getException()
     {
-        return m_excep;
+        return m_exception;
     }
 
     public boolean isSuccessful()
     {
         return m_success;
     }
 
-    private Exception m_excep = null;
+    private Exception m_exception = null;
     private RESULT m_result = null;
     private boolean m_success = true;
+    private String error_message = null;
```

Comment:
rename to m_errorMessage

Reply:
is this property missing a public getter?

Reply:
Thanks for pointing out the missing getter. Added!